app/javascript/packages/document-capture/components/in-person-outage-alert.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from '@18f/identity-i18n';
+import { Alert } from '@18f/identity-components';
+import { useContext } from 'react';
+import { getConfigValue } from '@18f/identity-config';
+import { InPersonContext } from '../context';
+
+function InPersonOutageAlert() {
+  const { inPersonOutageExpectedUpdateDate } = useContext(InPersonContext);
+
+  return (
+    <Alert type="warning" className="margin-bottom-4" textTag="div">
+      <>
+        <p className="margin-bottom-2">
+          <strong>
+            {t('idv.failure.exceptions.in_person_outage_error_message.post_cta.title', {
+              date: new Intl.DateTimeFormat(document.documentElement.lang, {
+                weekday: 'long',
+                month: 'long',
+                day: 'numeric',
+              }).format(new Date(`${inPersonOutageExpectedUpdateDate}`)),
```

Comment:
is the `${}` redundant here?
```suggestion
              }).format(new Date(inPersonOutageExpectedUpdateDate)),
```

Reply:
this was to account for when the date was undefined, I just made an update to check for that

Reply:
working on adding tests for the config value instead of doing an error check that allows the component to still render w/o the date if the config is empty. I'm wondering now how the specs run for each environment / will adding a test for the config value catch a missing value in prod? 
The error catching I currently have would allow the component to render w/o a date thus rendering the warning less useful. Seems like it should not fail silently/not render w/o a date. Wondering if adding this test is the best way to achieve that or maybe throwing an error instead?